ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -180,6 +180,22 @@ def min(*input, dim=None, keepdim=False, out=None):
         )
 
 
+@numpy_to_torch_style_args
+@to_ivy_arrays_and_back
+def mode(input, dim=-1, keepdim=False, *, out=None):
+    if dim is None:
+        input = ivy.reshape(input, [-1])
+        dim = 0
```

Comment:
we have a default value for dim parameter, so you don't need to check for None here.